src/main/java/com/xebialabs/overthere/util/WindowsCommandLineArgsSanitizer.java

```diff
@@ -39,6 +39,9 @@ private static String escapeSpecialCharacters(String str) {
         StringBuilder builder = new StringBuilder();
         for (int j = 0; j < str.length(); j++) {
             char c = str.charAt(j);
+            if (c == '%') {
```

Comment:
Why can't we add % into SPECIAL_CHARS and avoid using additional if?

Reply:
% can't be escaped with a ^. It's handled differently for some reason.
Source:
https://www.robvanderwoude.com/escapechars.php

Reply:
There are some other characters which are not present in our `SPECIAL_CHARS`, but those should only pose a problem in FOR and FIND commands. The funny thing is that most of those characters are also not escaped by a caret but by a backslash.